scripts/encrypter.rb

```diff
@@ -0,0 +1,6 @@
+require "android-all-4.4_r1-robolectric-0.jar"
```

Comment:
Can you verify your version of jruby is set to 1.7.12. Also this script should not be invoked directly rather through `encrypter.sh` which downloads the robolectric jar.